### ISSUE TITLE
The Getaway [BUG]:  Very slow FMVs with any renderer in any resolution but stable 60 FPS in 3x native  ingame

### ISSUE THREAD

**Issue by draccfacc** [PCSX2/pcsx2#9714]
### Describe the Bug

 FMVs are  very slow with any renderer in any resolution but stable 60 FPS in 3x native ingame.

### Reproduction Steps

1.Start new game
2.Watch the opening FMV (very slow)
3.After the gameplay starts it works full speed again

### Expected Behavior

FMVs work full speed as the gameplay does.

### PCSX2 Revision

1.7.4857

### Operating System

Windows 11

### CPU

R5 1600 AF

### GPU

R9 370X

### GS Settings

![3](https://github.com/PCSX2/pcsx2/assets/141240645/c7dabede-d766-4784-9fe8-42b69e894db2)
![4](https://github.com/PCSX2/pcsx2/assets/141240645/79997149-f4af-49c5-a6e2-d68f53d034b7)

### Logs & Dumps

![1](https://github.com/PCSX2/pcsx2/assets/141240645/e840da0e-bc6c-437d-a551-4849b51827fb)
![2](https://github.com/PCSX2/pcsx2/assets/141240645/a2870196-fb89-4af2-a0e2-69c9b390fd00)
![Getaway, The_SCUS-97133_20230803185131](https://github.com/PCSX2/pcsx2/assets/141240645/b0e11f81-89e1-431d-93f2-9f1aef5b83ff)
![Getaway, The_SCUS-97133_20230804131124](https://github.com/PCSX2/pcsx2/assets/141240645/efac8082-9f17-4ff6-9175-492ff2725e44)
[emulog.txt](https://github.com/PCSX2/pcsx2/files/12259673/emulog.txt)

**Comment by refractionpcsx2** [comment]
Yes this is a known issue, mostly caused by readbacks, iirc, since the FMV's are mostly realtime.

But since we don't have any issue open, I'll leave it here, in case we can ever improve it.

**Comment by stenzek** [comment]
I'd say close it - it's not a bug, and we already looked into this - the game downloads the framebuffer for its reflections and screws with it on the EE.

If you don't care about the reflections, you can disable hardware downloads.

**Comment by draccfacc** [comment]
> Yes this is a known issue, mostly caused by readbacks, iirc, since the FMV's are mostly realtime.
> 
> But since we don't have any issue open, I'll leave it here, in case we can ever improve it.

Can I ask question about the other game here?  It seems to be pretty much a similar situation in Ace combat 4 (Ingame 3x native - stable 60 fps and slowdowns in hangar mode)

**Comment by draccfacc** [comment]
> I'd say close it - it's not a bug, and we already looked into this - the game downloads the framebuffer for its reflections and screws with it on the EE.
> 
> If you don't care about the reflections, you can disable hardware downloads.

Reflections in FMVs? Cause it seems to be fullspeed in game as it is.

**Comment by refractionpcsx2** [comment]
I would appreciate it if you asked questions about going slow on the discord or forum first.

I will take Stens judgement on on this, since he is a GPU guy, if we can't really improve it, then we can't improve it, simple as that. sorry. You will have to switch temporarily to software mode (or get a more powerful machine).

**Comment by draccfacc** [comment]
> I would appreciate it if you asked questions about going slow on the discord or forum first.
> 
> I will take Stens judgement on on this, since he is a GPU guy, if we can't really improve it, then we can't improve it, simple as that. sorry. You will have to switch temporarily to software mode (or get a more powerful machine).

Swithing to SW makes no difference. As stated in the issue header.

**Comment by refractionpcsx2** [comment]
then the issue is your computer is too slow.

**Comment by stenzek** [comment]
It's extra heavy on HW not only because of the readbacks, but also because the game is drawing a point which doesn't show up on screen inbetween a good chunk of the triangles, which causes a flush. That's why it hits like 7000+ draw calls.

I have a branch which buffers draws to work around this, but it doesn't currently help getaway, because it also reloads the CLUT on these draws. So, really, all you can do for now is throw more hardware at it. My 5950X holds 70-80fps in SW.
